app/auth/utils.py

```diff
@@ -54,6 +60,24 @@ def user_loader(user_id):
     return Users.query.filter_by(guid=user_id[0], auth_user_type=user_id[1]).first()
 
 
+def destroy_session(user_id):
```

Comment:
This function is never called when logging out. I think you use "session.destroy()" instead.